pint/models/phase_jump.py

```diff
@@ -9,11 +9,14 @@
 
 
 class PhaseJump(TimingModel):
-    """This is a class to implement phase jumps 
+    """This is a class to implement phase jumps
     """
     def __init__(self):
         super(PhaseJump, self).__init__()
-        self.add_param(p.maskParameter(name = 'JUMP'))
+        # NOTE: it is dangours to put jump unit as second.
+        # Since JUMP is not always in the unit of time. 
```

Comment:
I'm confused here. When are JUMPs not in time units? I thought all JUMPs were time, or maybe I'm forgetting something.

Also, these are called `PhaseJump` and are implemented in the `phase_funcs` part of the timing model.  Is that the way TEMPO implements jumps?  What the the procedure for converting between the time units of the jump and phase?